data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,481 @@
+			`	The cadet takes you to an elevator just outside your ship, down more than three hundred floors to a large meeting room. Within, you see two men and a woman in formal dress. One man has a Republic Intelligence uniform with the name "Agent Kopkow". Another wears a Navy full dress uniform that you don't immediately recognize. After a moment of thought, you realize he is a Marshal, the second-highest rank in the Navy, answering only to the Grand Marshal. His uniform is labeled "Adurath Kaine". The woman is a Navy captain wearing the prominent Parliamentary Medal of Valor. Her name label says "Captain Anna Wallace."`
+			choice
+				`	(Speak to the agent and Marshal.)`
+					goto join
+				`	(Ruminate first.)`
+			`	You've had few dealings with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament.`
+			`	There are only five Navy Marshals, and one Grand Marshal; this man's high rank means he reports to Parliament regularly. People this high in the Navy tend to deal more with political issues rather than combat or strategy, so you wonder why they picked him to talk to you.`
+			`	The Parliamentary Medal of Valor is only given to one person per year; this captain must have done something especially heroic. She must also have supporters in Parliament to receive the award in the first place. That will give her some freedom to act independently, like Danforth does.`
+			`	The three of them are looking at you confused, wondering why you've stopped and stared at them for a half minute.`
+			choice
+				`	(Stop staring blankly and speak to them.)`
+			label join
+			`	Marshal Kaine introduces himself. "I'm the Marshal of the Third Armada, the guards of the Far North and the uninhabited regions near it. I know you've met one of my admirals, Danforth that is, a good man - albeit one who flouts the rules a bit too much." He gives a stern look at Captain Wallace. "Some of my carriers have detected the signals you've found. Actually, we see them quite regularly in the ships of known slave runners. The runners don't always carry slaves; they do other smuggling missions as well."`
+			`	Agent Kopkow speaks his piece next. "Sometimes the runners do carry slaves, and we have found humans or Hai in crashed slaver ships. We returned the Hai to their homes, as you did, but they didn't know Republic Intelligence helped them. Several merchants have done the same. About one in seven we rescued were researchers, which is an extremely disproportionate amount when you consider the number of researchers in human and Hai space. Clearly, they're being targeted."`
```

Comment:
F*** me. Did no one tell the Hai authorities about these kdinapped Hai found in human space???
*rages*